en/codelists-guides/codelist-management.rst

```diff
@@ -2,108 +2,115 @@ IATI Codelist Management
+.. _core_codelist:
 
-Embedded Codelists
+Embedded Codelists - Core
 ------------------
 
-These are 'functional' codelists that are core to IATI. Changing them
-has a big impact on all those publishing and using IATI data. For example, adding, removing
-or renaming something on the transaction-types codelist
-http://iatistandard.org/codelists/transaction_type/ could
-be very disruptive.
+Core Codelists contain conforming values for a particular **decimal Version** of the IATI Standard, the contents of which are managed by the IATI Technical Team. Core Codelists contain codes that involve functional logic that impacts the way in which the Standard is interpreted and processed.
 
-Embedded codelists are  managed via GitHub at:
+These are Codelists that are core to IATI: changing them has a big on impact on all those publishing and using IATI data.
+
+Embedded codelists are  managed via GitHub at:
 https://github.com/IATI/IATI-Codelists/
 
--  Any changes to these codelists should be handled through a formal
-   decimal or integer upgrade process.
-   (http://dev.iatistandard.org/upgrades/).
-   (except bug fixes - see below)
--  Releases are tagged in line with the IATI Standard versions. This
-   means you can grab a snapshot of e.g. everything to do with version
-   1.03.
--  There are branches for each version of the standard. These can change
-   between decimal and integer releases of the standard ONLY for bug
-   fixes, typos, and translations. Users can chose to track changes to
-   those branches if they like but our documentation (from v1.04
-   onwards) will always track the corresponding branch.
--  During an upgrade to the standard, the new version of a new branch is
-   created for development.
-
-.. _non_embedded_codelist:
-
-Non-Embedded Codelists
+**Modification Schedule: Embedded Codelists - Core**
+
+Codes in Core Codelists may be:
+
+- Added - at a Minor or Major Upgrade, through the applicable consultation process
+- Modified - at a Minor or Major Upgrade, through the applicable consultation process
+- Withdrawn - at a Minor or Major Upgrade, through the applicable consultation process
+- Removed - at a Major Upgrade, through the applicable consultation process
+
+See more information about the IATI Upgrade process here: http://reference.iatistandard.org/202/upgrades/
+
+Examples of Core Codelists include: Transaction Type, Organisation Type.
+
+.. _non_core_codelist:
+
+Non-Embedded Codelists - Non-Core
 ----------------------
 
-These 'non-functional' codelists usually provide lookup information on
-e.g. currencies, languages in use, countries, etc. Often they are
-derived from third party lists and an addition or modification has a
-minor impact on the sense and use of the data.
+Non-Core Codelists contain conforming values for a particular **integer** version of the IATI Standard, the contents of which are managed by the IATI Technical Team. Non-Core Codelists contain Codes that qualify data, not processes.
```

Comment:
> for a particular **integer** version of the IATI Standard

This seems like a significant change. Previously, all non-embedded codelists were version independent. Is this a mistake, or has this actually changed?

Reply:
Thanks @andylolz, yes that was a mistake on our side.